tidb-cloud/connect-to-tidb-cluster.md

```diff
@@ -43,7 +43,7 @@ To connect to a Serverless Tier cluster via standard connection, perform the fol
 
 To connect to a Dedicated Tier cluster via standard connection, perform the following steps:
 
-1. Navigate to the **Clusters** page.
+1. Navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page and choose your target project on the top of the left navigation bar.
 
 2. Locate your cluster, and click **Connect** in the upper-right corner of the cluster area. A connection dialog box is displayed.
 
```

Comment:
L61: The UI is "Update Filter"
<img width="1170" alt="image" src="https://user-images.githubusercontent.com/60599231/207009107-d5a15de2-ea73-4808-87a2-215294a11b00.png">

L68 "Currently, TiDB Cloud only provides the connection strings and sample code for these connection methods: MySQL, MyCLI, JDBC, Python, Go, and Node.js." is a note for step 5.